examples/unused_import/erc20_implement_pass.vy

```diff
@@ -0,0 +1,149 @@
+# @version ^0.3.7
+
+# @dev Implementation of ERC-20 token standard.
+# @author Takayuki Jimba (@yudetamago)
+# https://github.com/ethereum/EIPs/blob/master/EIPS/eip-20.md
+
```

Comment:
I would keep only relevant code in those examples, as you did everywhere else. Starting by all those comments, and I'll suggest other removals.
```suggestion
```

Reply:
This is from vyper's examples so I prefer leaving it as it is.